edge-modules/api-proxy-module/src/signals/shutdown.rs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+// Adapted from the conduit proxy signal handling:
+// https://github.com/runconduit/conduit/blob/master/proxy/src/signal.rs
+
+pub async fn shutdown() {
+    imp::shutdown().await
+}
+
+mod imp {
```

Comment:
since you don't need windows support, you can simply remove `imp` module and use its content as-is.

Reply:
Good catch, fixed.